runtime/makelib/targets.mk.aix.inc.ftl

```diff
@@ -99,31 +102,19 @@ ifdef I5_VERSION
   I5_FLAGS += -qtbtable=full -qlist -qsource
   I5_DEFINES += -DJ9OS_I5 -DJ9OS_$(I5_VERSION) -I$(UMA_PATH_TO_ROOT)/iseries
 
-  UMA_ASPP_DEBUG +=
   CFLAGS += $(I5_FLAGS) $(I5_DEFINES)
   CXXFLAGS += $(I5_FLAGS) $(I5_DEFINES)
   CPPFLAGS += $(I5_DEFINES)
 endif
 
-ifdef UMA_USING_LD_TO_LINK
-  ifeq ($(j9vm_env_data64),1)
-    UMA_DLL_LINK_FLAGS += -b64
-  else
-    UMA_DLL_LINK_FLAGS += -b32
-  endif
-  UMA_DLL_LINK_FLAGS += -G -bnoentry -bernotok
-  UMA_DLL_LINK_FLAGS += -bmap:$(UMA_TARGET_NAME).map
-  UMA_DLL_LINK_FLAGS += -bE:$(UMA_TARGET_NAME).exp
-  UMA_LINK_LIBRARIES += -lc_r -lC_r -lm -lpthread
+ifeq ($(j9vm_env_data64),1)
+  UMA_DLL_LINK_FLAGS += -q64
 else
-  ifeq ($(j9vm_env_data64),1)
-    UMA_DLL_LINK_FLAGS+=-X64
-  else
-    UMA_DLL_LINK_FLAGS+=-X32
-  endif
-  UMA_DLL_LINK_FLAGS+=-p 0 -brtl -G -bernotok -bnoentry -lm
-  UMA_DLL_LINK_FLAGS+=-E $(UMA_TARGET_NAME).exp
+  UMA_DLL_LINK_FLAGS += -q32
 endif
+# Link options like '-brtl', '-G', etc. must be prefixed by '-Wl,' to make xlc_r pass them through.
+UMA_DLL_LINK_FLAGS += -qmkshrobj -Wl,-brtl -Wl,-G -Wl,-bernotok -Wl,-bnoentry -lm
```

Comment:
Did the ` -bmap:$(UMA_TARGET_NAME).map` get deliberately dropped (why) or accidentally lost?

Reply:
I'll restore the map creation.